.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: K3Pomdav7dFeKalQEyIcFu1R9hoP007gg
```

Comment:
How else can we put the token in this repo? it is needed by `circle ci` to send the test reports to `coveralls`